ocean_lib/example_config.py

```diff
@@ -87,6 +87,12 @@
         NETWORK_NAME: "celoalfajores",
         NAME_BLOCK_CONFIRMATIONS: 3,
     },
+    80001: {
+        NAME_PROVIDER_URL: "https://provider.mumbai.oceanprotocol.com",
+        NAME_METADATA_CACHE_URI: "https://aquarius.oceanprotocol.com",
```

Comment:
@mariacarmina I'm thinking whether now, with universal aquarius, we can stop pasting this into the config. Can we set it independently?

Reply:
Ok, I have tested the marketplace flow and it works fine for ganache as well since it had the different aquarius URL.

Reply:
UPDATE: Created #525 to fix the duplication of the aquarius URL.